src/MVTLayer.js

```diff
@@ -169,6 +169,7 @@ module.exports = L.TileLayer.Canvas.extend({
     var toIndex = [];
     for (var i = 0, len = features.length; i < len; i++) {
       var vtf = features[i]; //vector tile feature
+      vtf.layer = vtl;
```

Comment:
This was needed to make the library examples work - figured out it'd be nice to have the original lib functionalities.
(as seen in https://github.com/SpatialServer/Leaflet.MapboxVectorTile/blob/master/src/MVTLayer.js#L170)